Makefile

```diff
@@ -54,7 +54,7 @@ functest: generate fmt vet manifests
 
 # Build manager binary
 manager: generate fmt vet
-	go build -o bin/manager main.go
+	go build -ldflags="-X 'kubevirt.io/ssp-operator/internal/operands/template-validator/template_validator.DefaultTemplateValidatorImage=${VALIDATOR_IMG}'" -o bin/manager main.go
```

Comment:
Can you extract the build command into a shell script? For example `hack/build-operator.sh`. Then call it here and from the Dockerfile?